Classes/Hook/UserSettings.php

```diff
@@ -152,7 +152,19 @@ private function getLayer(): string
             $layer = 'Backend';
         }
 
-        return $layer;
+        $dispatcher = GeneralUtility::makeInstance(Dispatcher::class);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Avoid using static access to class '\TYPO3\CMS\Core\Utility\GeneralUtility' in method 'getLayer'.](https://app.codacy.com/manual/robinvonberg/cf_google_authenticator/pullRequest?prid=5687502)